packages/terra-tag/src/terra-dev-site/test/tag/HrefTag.test.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-alert */
```

Comment:
Why was this disabled eslint rule added?

Reply:
Same as above.